dijet/tasks/base.py

```diff
@@ -57,8 +60,8 @@ def get_datasets(self) -> tuple[list[str], bool]:
         # check that at least one config dataset matched
         if not dataset_insts_from_process:
             raise RuntimeError(
-                "no single dataset found in config matching "
-                f"process `{self.branch_data.process}`"
+                "no single dataset found in config matching ",
```

Comment:
No comma needed here since the string continues on the next line.
```suggestion
                "no single dataset found in config matching "
```